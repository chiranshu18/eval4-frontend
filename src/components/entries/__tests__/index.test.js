import Entries from '../index';
import { render , screen} from '@testing-library/react';
import React from 'react';

describe('Entries', () => {
  it('Should render correctly', () => {
    const {asFragment} = render(<Entries />);
    expect(asFragment()).toMatchSnapshot();
  });
});