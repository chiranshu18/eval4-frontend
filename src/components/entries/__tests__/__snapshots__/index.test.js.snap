// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entries Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="entries"
  >
    <div
      class="entries-title"
    >
      <h1
        class="total-entries"
      >
        13 Entries Found
      </h1>
      <h2>
        Add New Entry
      </h2>
    </div>
    <div
      class="head-entry-card"
    >
      <div
        class="entry-id"
      >
        <p>
          id
        </p>
      </div>
      <div
        class="entry-name"
      >
        <p>
          name
        </p>
      </div>
      <div
        class="entry-website"
      >
        <p>
          website
        </p>
      </div>
      <div
        class="entry-contact"
      >
        <p>
          contact
        </p>
      </div>
      <div
        class="entry-actions"
      >
        actions
      </div>
    </div>
    <div
      class="all-entry-cards"
    >
      <div
        class="each-entry-card"
      >
        <div
          class="entry-id"
        >
          <p>
            1
          </p>
        </div>
        <div
          class="entry-name"
        >
          <p>
            name
          </p>
        </div>
        <div
          class="entry-website"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-contact"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-actions"
        >
          <img
            alt="delete"
            src="icon1.png"
          />
          <img
            alt="search"
            src="icon1.png"
          />
        </div>
      </div>
      <div
        class="each-entry-card"
      >
        <div
          class="entry-id"
        >
          <p>
            1
          </p>
        </div>
        <div
          class="entry-name"
        >
          <p>
            name
          </p>
        </div>
        <div
          class="entry-website"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-contact"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-actions"
        >
          <img
            alt="delete"
            src="icon1.png"
          />
          <img
            alt="search"
            src="icon1.png"
          />
        </div>
      </div>
      <div
        class="each-entry-card"
      >
        <div
          class="entry-id"
        >
          <p>
            1
          </p>
        </div>
        <div
          class="entry-name"
        >
          <p>
            name
          </p>
        </div>
        <div
          class="entry-website"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-contact"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-actions"
        >
          <img
            alt="delete"
            src="icon1.png"
          />
          <img
            alt="search"
            src="icon1.png"
          />
        </div>
      </div>
      <div
        class="each-entry-card"
      >
        <div
          class="entry-id"
        >
          <p>
            1
          </p>
        </div>
        <div
          class="entry-name"
        >
          <p>
            name
          </p>
        </div>
        <div
          class="entry-website"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-contact"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-actions"
        >
          <img
            alt="delete"
            src="icon1.png"
          />
          <img
            alt="search"
            src="icon1.png"
          />
        </div>
      </div>
      <div
        class="each-entry-card"
      >
        <div
          class="entry-id"
        >
          <p>
            1
          </p>
        </div>
        <div
          class="entry-name"
        >
          <p>
            name
          </p>
        </div>
        <div
          class="entry-website"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-contact"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-actions"
        >
          <img
            alt="delete"
            src="icon1.png"
          />
          <img
            alt="search"
            src="icon1.png"
          />
        </div>
      </div>
      <div
        class="each-entry-card"
      >
        <div
          class="entry-id"
        >
          <p>
            1
          </p>
        </div>
        <div
          class="entry-name"
        >
          <p>
            name
          </p>
        </div>
        <div
          class="entry-website"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-contact"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-actions"
        >
          <img
            alt="delete"
            src="icon1.png"
          />
          <img
            alt="search"
            src="icon1.png"
          />
        </div>
      </div>
      <div
        class="each-entry-card"
      >
        <div
          class="entry-id"
        >
          <p>
            1
          </p>
        </div>
        <div
          class="entry-name"
        >
          <p>
            name
          </p>
        </div>
        <div
          class="entry-website"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-contact"
        >
          <p>
            text
          </p>
        </div>
        <div
          class="entry-actions"
        >
          <img
            alt="delete"
            src="icon1.png"
          />
          <img
            alt="search"
            src="icon1.png"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
